src/common/procedure/src/local/runner.rs

```diff
@@ -472,7 +473,7 @@ mod tests {
     }
 
     async fn check_files(object_store: &ObjectStore, procedure_id: ProcedureId, files: &[&str]) {
-        let dir = format!("{procedure_id}/");
+        let dir = format!("{PROC_PATH}/{procedure_id}/");
```

Comment:
use `proc_path!` too?

Reply:
This is in another mod and is unable to reuse the macro defined in `store.rs` directly.